nova_vm/Cargo.toml

```diff
@@ -20,6 +20,7 @@ rand = { workspace = true }
 ryu-js = { workspace = true }
 small_string = { path = "../small_string" }
 sonic-rs = { workspace = true, optional = true }
+unicode-normalization = "0.1.24"
```

Comment:
**issue:** We've been keeping all our dependencies at the top level, so please add this dependency to the main `Cargo.toml` and then make the dependency `{ workspace = true }` here.

Reply:
Got it, it's now moved at the top level.